### PR TITLE
[qwik-pod] feat: user logout + export types

### PR DESCRIPTION
**Pull request by honzikec**
- Added logout functionality
- Exported types in central place
- Improved some stories

### CODE REVIEW

File: qwik-graphql-tailwind/src/routes/repositories/index.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { component$, useClientEffect$, useStore } from '@builder.io/qwik';
+import { AUTH_TOKEN, GITHUB_GRAPHQL } from '~/utils/constants';
+import { REPOS_QUERY } from '~/utils/queries/repos-query';
+import { TOP_REPOS_QUERY } from '~/utils/queries/top-repos-query';
```
Comment (AllStackDev1) [inline qwik-graphql-tailwind/src/routes/repositories/index.tsx:4]: This page doesn't exist anymore

**Review by AllStackDev1 — CHANGES_REQUESTED** [review]
Need to remove files/folders we don't make use of anymore eg; the repositories

File: qwik-graphql-tailwind/src/routes/repositories/index.tsx (hunk 1 above)
Comment (SDaian) [reply qwik-graphql-tailwind/src/routes/repositories/index.tsx]: removed

**Review by AllStackDev1 — APPROVED** [review]

**Review by AllStackDev1 — APPROVED** [review]
LGTM

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx [hunk 2]
```diff
@@ -24,8 +25,17 @@ export const UserDropdown = component$(({ image, username }: UserDropdownProps)
     store.expanded = false;
   });
 
-  const signOut$ = $(() => {
-    // TODO: sign out
+  const signOut$ = $(async () => {
+    sessionStorage.removeItem(AUTH_TOKEN);
+    fetch(SIGN_OUT_URL, {
```
Comment (hdJerry) [inline qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:30]: why do we have to do this please?

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx [hunk 3]
```diff
@@ -24,8 +25,17 @@ export const UserDropdown = component$(({ image, username }: UserDropdownProps)
...
+      method: 'POST',
+    })
+      .then(() => {
+        window.location.href = '/auth/signin';
```
Comment (hdJerry) [inline qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:34]: is this the only way qwik offers for now?

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx (hunk 2 above)
Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:30]: This is a feature of Qwik, [here](https://qwik.builder.io/docs/advanced/optimizer/#the-meaning-of-)

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx (hunk 3 above)
Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:34]: This is a feature of Qwik, [here](https://qwik.builder.io/docs/advanced/optimizer/#the-meaning-of-)

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx (hunk 2 above)
Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:30]: Can you elaborate on this? what exactly is the issue

File: qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx (hunk 3 above)
Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:34]: are you talking about the fetch function and the redirection?

Comment (AllStackDev1) [reply qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx:34]: The only thing qwik here is the $(), and I believe you now know what it is for.

**Pull request merged**